docs/en_US/install/wiiu.md

```diff
@@ -90,3 +92,26 @@ Right before registering a PNID, please make sure to save your password on some
 <img src="/assets/images/docs/install/wiiu/pnid/pretendo-eula.jpg" width="100%">
 
 Finally, set up your PNID, email and password. Once all of this is done, you will be able to use Pretendo servers.
+
+# Switching between Pretendo Network and Nintendo Network
+
+Occasionally, you may want to switch between your Pretendo Network and your Nintendo Network profiles on your Wii U; fortunately, it is super simple to switch between the two.
+
+While connected to Pretendo, in the HOME Menu, switch to your Nintendo Network profile. If prompted for a password, do not provide it (if you don't have auto sign-in enabled, you may not have the option to not provide it). You are now inside your Nintendo Network profile.
+
+On your Wii U GamePad, press the buttons `L + Down + SELECT`. This action opens up the Aroma plugin menu, which should look something like this:
+
+<img src="/assets/images/docs/install/wiiu/pnid/aroma-plugins-list.jpg" width="100%">
+
+Your menu might display additional plugins that aren't shown in this image; however, our focus is specifically on the 'Inkay' plugin, which patches requests intended for Nintendo Network and reroutes them to Pretendo Network instead. Once you've found the entry for Inkay, press `A`.
```

Comment:
I'm gonna use this paragraph as an example but this comment goes for basically the whole page

While I do enjoy having a fun/"personal" vibe with our brand generally, when it comes to our official documentation I do prefer to keep it as short, concise, and professional as possible. That will likely make the documentation come off as "corpo", but there are tons of people who won't understand the instructions unless it's handed to them in the simplest way possible

Doing things this way avoids sections that get too wordy with no real benefit to the user, and gets directly to the point of the section

There's no hard set of rules or guidelines for this though. I kinda just judge if based off vibes. Like for this section I would suggest something like

```suggestion
You may have additional plugins in your menu. Navigate to the `Inkay` plugin and press `A` to enter it's settings.
```

This removes a good portion of fluff from this section, mostly related to the fact that it's already clear what Inkay does based on information earlier in the page. That doesn't need to be restated

Reply:
Ok that's also on me. whoops

Reply:
It's not just your changes I actually looked at this PR when it was made and wanted to say the same thing then, I just didn't actually make a review at the time